scripts/apps/search/styles/search.scss

```diff
@@ -541,4 +541,17 @@
 
         }
     }
+}
+
+
+// used to resize thumbnail of item while dragging it
+#drag-image-holder {
+    position: absolute;
+    left: -2000px;
+    float: left;
+    max-width: 200px;
+
+    img {
```

Comment:
Can you post a gif to illustrate what's the problem with dragging? It'd be nice not to rely upon certain elements being inside. Can't a clone of the element be displayed while dragging or doesn't it work for us for some reason?

Reply:
seems like I can't do it, that dragged image is not visible while doing screen grabbing or screen shot. but you can see the big pictures [here](https://bel-master.test.superdesk.org/#/search?repo=5cac5110405ecc00e4403708). clone doesn't work, it would always use the full size of image, that's why container is needed

Reply:
@tomaskikutis so did a screenshot with phone..
before:
![DSC_0049](https://user-images.githubusercontent.com/179249/75433556-6b428a00-5950-11ea-9d72-d4478915c6c7.JPG)
after:
![DSC_0050](https://user-images.githubusercontent.com/179249/75433578-739ac500-5950-11ea-8048-33210ccc65ee.JPG)

Reply:
That's weird. I've tried recording it with the simple screen recorder and it did appear in the video for me.

Reply:
I guess it wasn't on debian..

Reply:
and it works on ubuntu I have on other PC actually, probably gnome/wayland version problem